FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -4,9 +4,20 @@
 
 import Foundation
 
+public struct FeedImageListRemote: Decodable {
+	public let items: [FeedImageRemote]
+}
+
+public struct FeedImageRemote: Decodable {
+	public let imageId: UUID
+	public let imageDesc: String?
+	public let imageLoc: String?
+	public let imageUrl: URL
+}
+
 public final class RemoteFeedLoader: FeedLoader {
 	private let url: URL
-	private let client: HTTPClient
+	private var client: HTTPClient
```

Comment:
This code should not be changed, please revert the change

```suggestion
	private let client: HTTPClient
```